File-Formats/Presentation/Presentation-to-image.md

```diff
@@ -192,6 +198,113 @@ pptxDoc.Close()
 
 {% endtabs %}
 
+## Convert PowerPoint Animations to Images
+
+The .NET PowerPoint Library (Presentation) allows you to convert PowerPoint slides into images based on the sequence of entrance animation effects applied to each element.
+
+For instance, if a slide includes bulleted paragraphs, each having entrance animation effects, the Presentation library converts every bulleted paragraph into a separate image.
+
+N> Only entrance animation effects are considered for generating separate images. Other animation effects and non-animated elements will be converted into images within the first slide. 
```

Comment:
N> Only the entrance animation effects are considered for generating separate images. Other animation effects and non-animated elements will be converted into images within the first slide.